static/js/ktl-component/why-teach/index.jsx

```diff
@@ -0,0 +1,663 @@
+                  <p className="ktl-text-2 ktl-offset-bottom-s ktl-dimmed-text">
+                    of the top 100 universities in the Times Higher Education World University Rankings 2021 include
+                    Kotlin
+                    in
+                    their courses.
+                  </p>
+
+                  <a
+                    href="https://docs.google.com/spreadsheets/d/1p77WHo--mxewmxINWMLaTPGXvnEr0JGxgSMcX6C0b_0/edit?usp=sharing"
+                    target="_blank" className="ktl-text-2 ktl-link">
+                    <span className="rs-link">List of universities</span>
+                    <span>&nbsp;↗</span>
+                  </a>
+                </div>
+              </div>
+            </section>
+
+
+            <section className="quote-section ktl-offset-bottom-xl ktl-with-anchor" id="language-of-the-industry">
+              <h3 className="ktl-h3 ktl-offset-bottom-s">
+                Language of the industry
+              </h3>
+
+              <div className="quote-section__grid">
+                <div className="quote-section__content">
+
+                  <p className="ktl-text-1 ktl-offset-bottom-m">
+                    <ul>
+                      <li className="ktl-offset-bottom-s">
+                        — Kotlin is used by top companies such as Google, Twitter, Reddit, Netflix, Uber, BMW, Coursera,
+                        Slack, and Trello, just to name a few.
+                      </li>
+                      <li className="ktl-offset-bottom-s">
+                        — Kotlin has consistently ranked among the top 4 most-loved programming languages since 2018,
+                        according to the <a className="rs-link"
+                                            href="https://insights.stackoverflow.com/survey/2019#most-loved-dreaded-and-wanted">Stack
+                        Overflow Developer Surveys.</a>
+                      </li>
+
+                      <li className="ktl-offset-bottom-s">
+                        — Kotlin is one of the fastest-growing programming languages, ranking fourth in that category
+                        in <a
+                        className="rs-link" href="https://octoverse.github.com/">GitHub’s 2019 State of the Octoverse
+                        survey.</a>
+                      </li>
+
+                      <li>
+                        — Kotlin has the fastest growing language community. (<a className="rs-link"
+                                                                                 href="https://developer-economics.cdn.prismic.io/developer-economics/dbf9f36f-a31a-440a-9c22-c599cc235fa4_20th+edition+-+State+of+the+developer+Nation.pdf">SlashData's
+                        State of the Developer Nation 20th
+                        edition, Q1 2021</a>)
+                      </li>
+                    </ul>
+                  </p>
+
+                  <QuotesSlider quotes={[
```

Comment:
Top module constant for complex object is better for time and potentially better for render optimization... But inline is really better for reading. 🤔

@nikpachoo, @krutilov,, what do you think about use constant complex objects props?

Reply:
My initial thought was to move data into object, but as you mention readability is much better with inline data, so i stick with this approach

Reply:
For me is preferable to separate data from template:
- easy to read template without bunch of data
- template changes will not affect the data (for example remove or add extra parent div will affect data indent too)

Reply:
Moved to json